packages/commerce-sdk-react/src/auth/index.ts

```diff
@@ -179,7 +179,7 @@ class Auth {
     private clientSecret: string
     private silenceWarnings: boolean
     private logger: Logger
-    private defaultDnt: boolean | undefined
+    defaultDnt: boolean | undefined
```

Comment:
why removing `private` key word here?

Reply:
This is because `defaultDnt` has to be accessed outside of Auth scope in `updateDNT` to make sure the `defaultDnt` value is being used for `dw_dnt` cookie